repos/spack_repo/builtin/packages/kokkos_fft/package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack.package import *
+
+
+class KokkosFft(CMakePackage):
+    """FFT interfaces for Kokkos C++ Performance Portability Programming EcoSystem"""
+
+    homepage = "https://github.com/kokkos/kokkos-fft"
+    url = "https://github.com/kokkos/kokkos-fft/archive/refs/tags/v0.3.0.tar.gz"
+
+    maintainers("cedricchevalier19", "tpadioleau", "yasahi-hpc")
+
+    license("Apache-2.0 WITH LLVM-exception OR MIT", checked_by="cedricchevalier19")
+
+    version("0.3.0", sha256="a13c423775afec5f9f79fa9a23dd6001d3d63bae9f4786b1e0cd3ed65b3993a3")
+
+    variant("fftw", default=True, description="Enable FFTW")
+    variant("cufft", default=False, description="Enable cufft")
+    variant("hipfft", default=False, description="Enable hipfft")
+    variant("onemkl", default=False, description="Enable oneMKL fft")
+    variant("tests", default=False, description="Enable tests")
+
+    depends_on("kokkos@4.4:4 +complex_align") # not supported in Spack 0.23.1
```

Comment:
```suggestion
    depends_on("kokkos@4.4:4 +complex_align")
```

I do not think we'll have to deal with spack < 1.0

Reply:
Adastra is using Spack 0.23 but I guess we will need to backport this recipe elsewhere anyway.

Reply:
done in 3c3a83a9f7701bb7a0cc6e6d111ea23cf6a0adad